frontend/app/routing.js

```diff
@@ -79,7 +133,7 @@ angular.module('openproject')
       templateUrl: '/templates/work_packages.list.new.html'
     })
     .state('work-packages.list.details', {
-      url: '/{workPackageId:[0-9]+}?query_props',
+      url: '/details/{workPackageId:[0-9]+}?query_props',
```

Comment:
This change will break existing links people might already have stored outside of the system, .e.g by sending it via mail. Why is it necessary? The former route seemed to be uniq to me as well.